src/main/java/com/github/cloudyrock/mongock/Mongock.java

```diff
@@ -110,6 +112,18 @@ void execute() throws MongockException {
 
   }
 
+  Object o;
+  private Object getObject() {
+    if (o == null) {
+      synchronized (this) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this dangerous instance of double-checked locking. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2168)